packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/base-types.ts

```diff
@@ -991,3 +991,38 @@ class StandardS3Location extends S3Location {
     return { uri: this.uri };
   }
 }
+
+/**
+ * ML compute instance type for the transform job.
+ *
+ * @experimental
```

Comment:
i believe we dropped marking items as `@experimental` in a stable module and there should be a linter rule that will prevent this. Let's drop it as it was done in #14071 to prep us for v2.

Reply:
I see, fixed